src/Compat/CompatForm.php

```diff
@@ -12,6 +14,16 @@ class CompatForm extends Form
 
     protected $defaultAttributes = ['class' => 'icinga-form icinga-controls'];
 
+    public function renderContent(): string
+    {
+        if (count($this->submitElements) > 1) {
+            return (new HtmlDocument())
+                ->setHtmlContent($this->createSubmitButton($this->submitButton), new HtmlString(parent::renderContent()));
```

Comment:
Isn't this incompatible with the return type hint?

It may be implicitly casted, but I think calling `render()` explicitly is the better approach.